content/influxdb/cloud/organizations/migrate-org.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Migrate an organization
+seotitle: Migrate an organization in InfluxDB Cloud
+description: Replicate the state of an organization in InfluxDB Cloud
+menu:
+  influxdb_cloud:
+    name: Migrate organization
+    parent: Manage organizations
+weight: 110
+---
+
+The state of an consists of metadata (dashboards, buckets, and other resources) and data (time-series).
+While there no way to automatically duplicate an organization,
+the state of an organization can be migrated to a new organization.
```

Comment:
I'm not sure this is correct. Migrating the entire metadata and data state of an organization is effectively duplicating the org. Maybe something along the lines of:

An organization's state at a point in time can be effectively duplicated to another organization by copying both metadata and data. Copying only metadata can be useful for replicating the state of an organization for different environments, e.g. dev and prod. Copying only data can be useful for saving high value data to a location not subject to a retention policy.